src/js/components/Select/__tests__/__snapshots__/Select-multiple-test.js.snap

```diff
@@ -737,6 +737,15 @@ exports[`Select Controlled multiple onChange toggle with valueKey reduce 1`] = `
   width: 100%;
 }
 
+.c7 {
```

Comment:
Seems strange that we are getting snapshot changes for Select and SelectMultiple

Reply:
Agreed, although I skimmed through them and didn't see any that seemed broken, just re-ordered CSS selectors.